payments-core/src/main/java/com/stripe/android/GooglePayJsonFactory.kt

```diff
@@ -473,7 +473,7 @@ class GooglePayJsonFactory constructor(
 
         private val ALLOWED_AUTH_METHODS = listOf("PAN_ONLY", "CRYPTOGRAM_3DS")
         private val DEFAULT_CARD_NETWORKS =
-            listOf("AMEX", "DISCOVER", "MASTERCARD", "VISA")
+            listOf("AMEX", "DISCOVER", "MASTERCARD", "VISA", "MAESTRO")
```

Comment:
@samer-stripe Just confirming this should be a default vs. what we do with JCB?

Reply:
This should be a default yes. Maestro is available in multiple countries and is enabled by default on iOS as well.